llm-models/scripts/templates/text_generation/01_load_inference.py.jinja

```diff
@@ -0,0 +1,295 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # `{{ model_name }}` Inference on Databricks
+# MAGIC
+# MAGIC The [{{ fine_tuned_model_name }}](https://huggingface.co/{{ hf_org_name }}/{{ fine_tuned_model_name }}) Large Language Model (LLM) is a instruct fine-tuned version of the [{{ base_model_name }}](https://huggingface.co/{{ hf_org_name }}/{{ base_model_name }}) generative text model using a variety of publicly available conversation datasets.
+# MAGIC
+{% if support_vllm %}
+# MAGIC [vllm](https://github.com/vllm-project/vllm/tree/main) is an open-source library that makes LLM inference fast with various optimizations.
+{% endif %}
+# MAGIC Environment for this notebook:
+# MAGIC - Runtime: 14.2 GPU ML Runtime
+# MAGIC - Instance:
+{% for (cloud_name, compute_size) in compute_type.items() %}
+# MAGIC     - {{ compute_size }} on {{ cloud_name }}
+{% endfor %}
+
+{% if model_family_name == "llama-2"%}
+# MAGIC Requirements:
+# MAGIC - To get the access of the model on HuggingFace, please visit the [Meta website](https://ai.meta.com/resources/models-and-libraries/llama-downloads) and accept our license terms and acceptable use policy before submitting this form. Requests will be processed in 1-2 days.
+{% endif %}
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Install required packages
+
+# COMMAND ----------
+
+# MAGIC %pip install -U {% for pip_package in pip_requirements %} {{ pip_package }} {% endfor %}
+
+{% if support_vllm %}
+# MAGIC %pip install vllm
+{% endif %}
+# MAGIC dbutils.library.restartPython()
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Inference
+# MAGIC Load and run inference on Databricks.
+
+# COMMAND ----------
+from transformers import AutoTokenizer
+
+# Load the model
+
+# it is suggested to pin the revision commit hash and not change it for reproducibility because the uploader might change the model afterwards; you can find the commmit history of `{{ fine_tuned_model_name }}`. in https://huggingface.co/{{hf_org_name}}/{{fine_tuned_model_name}}/commits/main
+model = "{{ hf_org_name }}/{{ fine_tuned_model_name }}"
+revision = "{{ revision }}"
+
+tokenizer = AutoTokenizer.from_pretrained(model, padding_side="left")
```

Comment:
This line can be removed, as `tokenizer = ...` is also in the `{% else %}` block and the `{% if support_vllm %}` block doesn't use it